src/transformers/models/mistral/modeling_mistral.py

```diff
@@ -656,28 +656,34 @@ def forward(
         value_states = value_states.view(bsz, q_len, self.num_key_value_heads, self.head_dim).transpose(1, 2)
 
         kv_seq_len = key_states.shape[-2]
+        past_key_value = getattr(self, "past_key_value", past_key_value)
```

Comment:
Why the FA2 modules are not updated here whereas the FA2 module is being updated on Llama?

Reply:
Forgot about it! Thanks for catching